packages/@o3r/eslint-config-otter/package.json

```diff
@@ -63,7 +63,7 @@
     "@typescript-eslint/eslint-plugin": "^7.14.1",
     "@typescript-eslint/parser": "^7.14.1",
     "@typescript-eslint/utils": "^7.14.1",
-    "eslint": "^8.57.0",
+    "eslint": "^8.57.0 || ^9.0.0",
```

Comment:
eslint 9 is not supported for this package, right ?
is this linked to the version-harmonize ?